docs/client/components/pages/InlineToolbar/CustomInlineToolbarEditor/index.js

```diff
@@ -47,6 +47,10 @@ class HeadlinesPicker extends Component {
 }
 
 class HeadlinesButton extends Component {
+  // When using a click event inside overriden content we have
+  // to cancel click events from the toolbar itself unfortunately
```

Comment:
There's a small typo: "overridden"

Since this comment is in the docs and is supposed to educate users, what about something like: "When using a click event inside overridden content, mouse down events needs to be prevented so the focus stays in the editor and the toolbar remains visible."?

Reply:
@juliankrispel Don't you think this should be updated?

Reply:
@amann I did :) - you're looking at the outdated version ;)

Reply:
@amannn are you sure you don't want to join our fortnightly meetings?  ;)

Reply:
Ah – sorry, my bad! Haha, I'll think it over ;-)